.github/workflows/ci-main.yml

```diff
@@ -121,8 +121,9 @@ jobs:
         if: ${{ github.event_name == 'push' }}
         uses: ./.github/composites/docs-publish
         with:
-          aws-role: ${{ secrets.DOC_PUBLISH_ROLE }}
-
+          aws-role: ${{ secrets.DOCS_PUBLISH_ROLE }}
```

Comment:
Ugh good eye, that's a rough one. I was wondering if we'd be able to catch this with some form of linting, but afaict https://github.com/rhysd/actionlint could only assume that secret was defined and inherited in this case :-/